roles/icpc_host_backup/tasks/main.yml

```diff
@@ -35,41 +35,43 @@
 
 - name: Copy per contest hosts to /etc/hosts
   ansible.builtin.copy:
-    src: "hosts.{{ contest }}.new"
+    src: "files/hosts.{{ contest }}.new"
     dest: /etc/hosts
     mode: '0644'
   when: per_contest_hosts.stat.exists
 
 - name: Copy hosts to /etc/hosts
   ansible.builtin.copy:
-    src: hosts.new
+    src: files/hosts.new
     dest: /etc/hosts
     mode: '0644'
   when: not per_contest_hosts.stat.exists
 
-- name: Update ntp.conf
-  ansible.builtin.copy:
-    src: ntp.conf.new
-    dest: /etc/ntp.conf
-    mode: '0644'
-  notify:
-    - NTP
+# we use systemd-timesyncd instead
+# - name: Update ntp.conf
+#   ansible.builtin.copy:
+#     src: files/ntp.conf.new
+#     dest: /etc/ntp.conf
+#     mode: '0644'
+#   notify:
+#     - NTP
 
 - name: Install rsyslog.d config
   ansible.builtin.copy:
-    src: 40-backup.conf
+    src: files/40-backup.conf
     dest: /etc/rsyslog.d/40-backup.conf
     mode: '0644'
   notify:
     - Rsyslog
 
-- name: Install logrotate.d config
-  ansible.builtin.copy:
-    src: ZZicpc
-    dest: /etc/logrotate.d/ZZicpc
-    mode: '0644'
-  notify:
-    - Logrotate
+# TODO: this ruleset is bad/conflicts with /etc/logrotate.d/rsyslog for most of it's files
+# - name: Install logrotate.d config
+#   ansible.builtin.copy:
+#     src: files/ZZicpc
+#     dest: /etc/logrotate.d/ZZicpc
+#     mode: '0644'
+#   notify:
+#     - Logrotate
```

Comment:
what are we doing about changing the config to do daily instead of weekly?

Reply:
I guess we need to update the `ZZicpc` file to be correct, but I wasn't sure what it should be/just wanted to get the playbook functional.

Reply:
or update the existing config with lineinfile ?

Reply:
I think I'd rather overwrite than `lineinfile` it, just because it becomes more clear what we want. What do we actually care about with regards to logrotation?

Reply:
i mean if it's just to replace 
```
weekly
```
with 
```
daily
```
lineinfile is okay.

also the ZZicpc changed to include just:
```
/var/log/iptables.log
```

Reply:
There's a few other changes, I think it's ok to just ship the whole file for now. I'll push that change shortly.